From fa032acd93b7a474f81ba97a1e84dc908279f706 Mon Sep 17 00:00:00 2001
From: Jerome Martinez <jerome@mediaarea.net>
Date: Wed, 4 Jan 2023 14:32:46 +0100
Subject: [PATCH 02/19] libavcodec/flacenc: force 32-bit output for 32-bit
 input (no downscale)

Remove the need of -strict experimental for having a 32-bit output
---
 libavcodec/flacenc.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/libavcodec/flacenc.c b/libavcodec/flacenc.c
index ead2c55f10..03a22ba838 100644
--- a/libavcodec/flacenc.c
+++ b/libavcodec/flacenc.c
@@ -288,13 +288,6 @@ static av_cold int flac_encode_init(AVCodecContext *avctx)
                 av_log(avctx, AV_LOG_WARNING, "encoding as 24 bits-per-sample\n");
             avctx->bits_per_raw_sample = 24;
             s->bps_code                = 6;
-        } else if (avctx->strict_std_compliance > FF_COMPLIANCE_EXPERIMENTAL) {
-            av_log(avctx, AV_LOG_WARNING,
-                   "encoding as 24 bits-per-sample, more is considered "
-                   "experimental. Add -strict experimental if you want "
-                   "to encode more than 24 bits-per-sample\n");
-            avctx->bits_per_raw_sample = 24;
-            s->bps_code                = 6;
         } else {
             avctx->bits_per_raw_sample = 32;
             s->bps_code = 7;
-- 
2.52.0

